charts/invenio/templates/invenio-configmap.yaml

```diff
@@ -20,5 +20,8 @@ data:
   INVENIO_DATACITE_ENABLED: "False"
   INVENIO_LOGGING_CONSOLE_LEVEL: "WARNING"
   {{- range $key, $value := .Values.invenio.extra_config }}
-  {{ $key }}: {{ $value | toYaml | indent 4 | trim }}
+  {{ $key }}: {{ tpl $value $ | toYaml | indent 5 | trim }}
```

Comment:
I'm leaning towards that it's OK to run the `extraConfig` values through `tpl`. I'm guessing there won't be any issues with that, especially since you have documented that the values are templated.

But if we are so careful with the old `extra_config` value as to print a deprecation warning, then I think we should not change its behavior at all.

So basically:
- I'm good with templating the strings from `extraConfig` (unless someone can show me an example of where it breaks).
- Let's keep the `extra_config` values non-templated.

Does this work for you?

Reply:
That makes sense.